generators/app/templates/infrastructure/src/prisma/client.js

```diff
@@ -59,7 +59,7 @@ function prisma() {
         prismaClient.$use(async (params, next) => {
           const tableHasColumnTenant = tableHasColumnTenantId(params.model)
           if (tableHasColumnTenant) {
-            const { id } = getTenantContext()
+            const { id } = tenantContextAccessor.getTenantContext()
```

Comment:
This should be removed because you have this info at line 47

Reply:
done